panel/pane/holoviews.py

```diff
@@ -163,24 +164,38 @@ def _update_layout(self):
         layout, align, widget_first = alignments[loc]
         self.widget_box.align = align
         self._widget_container = self.widget_box
+        if self._width_responsive and self._height_responsive:
+            smode = 'stretch_both'
+        elif self._width_responsive:
+            smode = 'stretch_width'
+        elif self._height_responsive:
+            smode = 'stretch_height'
+        else:
+            smode = None
         if not len(self.widget_box):
             if center:
                 components = [HSpacer(), self, HSpacer()]
             else:
                 components = [self]
             self.layout[:] = components
+            self.layout.sizing_mode = smode
             return
 
         items = (self.widget_box, self) if widget_first else (self, self.widget_box)
+        kwargs = {'sizing_mode': smode}
         if not center:
-            components = [layout(*items)]
+            if False:#self.default_layout is layout:
```

Comment:
This looks like a debug thing

Reply:
Indeed, thanks for catching, may re-enable.